dap/dap.hpp

```diff
@@ -566,9 +569,211 @@ struct WXDLLIMPEXP_DAP InitializeRequest : public Request {
     JSON_SERIALIZE();
 };
 
+#if __cplusplus >= 201703L  // AKA C++17
+/// A ColumnDescriptor specifies what module attribute to show in a column of the ModulesView,
+/// how to format it, and what the column’s label should be.
+/// It is only used if the underlying UI actually supports this level of customization.
+struct ColumnDescriptor {
+    // Name of the attribute rendered in this column.
+    wxString attributeName;
+
+    // Header UI label of column.
+    wxString label;
+
+    // Format to use for the rendered values in this column. TBD how the format
+    // strings looks like.
+    std::optional<wxString> format;
```

Comment:
can't we use `wxAny` ? with value set to null to indicate "does not exist"

Reply:
I will check this out as I did not even look at wxAny. 

Thanks as this is exactly the feedback I was after.

Reply:
Changes pushed to change to use wxAny.